<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="KeyLineEdit" name="alt1"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelAlt1">
      <property name="text">
       <string>Alt+1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="KeyLineEdit" name="alt2"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelAlt2">
      <property name="text">
       <string>Alt+2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="KeyLineEdit" name="alt3"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelAlt3">
      <property name="text">
       <string>Alt+3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KeyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>keylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
